API/Login.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+	$inData = getRequestInfo();
+
+	$servername = "localhost";
+	$username = "root";
+	$password = "";
+	$dbname = "myDB";
+
+	// Create connection
+	$conn = new mysqli($servername, $username, $password, $dbname); 	
+	if( $conn->connect_error )
+	{
+		returnWithError( $conn->connect_error );
+	}
+	else
+	{
+		$userExists = userExists($conn, $inData['userName']);
+
+		if ($userExists === false)
+			returnWithError("User doesn't exist.")
+
+		$checkPass = password_verify($inData['password'], $userExists['password']);
+
+		if ($checkPass === false)
+			returnWithError("Incorrect password.");
+
+		returnWithInfo( $userExists['firstName'], $userExists['lastName'], $userExists['userID'] );
+
+		$stmt->close();
+		$conn->close();
+	}
+
+	function userExists($conn, $username) {
+		$stmt = $conn->prepare("SELECT * FROM Users WHERE userName = ?;");
```

Comment:
No need for ";" in statement
```suggestion
		$stmt = $conn->prepare("SELECT * FROM Users WHERE userName = ?");
```